doc/getting_started.rst

```diff
@@ -117,6 +117,17 @@ the test data::
   >>> accuracy_score(pipe.predict(X_test), y_test)
   0.97...
 
+.. note:: Diagram rendering of estimators
+
+    To render estimators as diagrams in notebooks, use the `display='diagram'` option:
+    >>> with config_context(display='diagram'):
+    >>>   pipe
```

Comment:
```suggestion
    To render estimators as diagrams in notebooks, use the `display='diagram'` option::
    
    >>> with config_context(display='diagram'):
    >>>     pipe
```

Reply:
(we need the `::` otherwise this won't be interpreted as code)

Reply:
@NicolasHug I accepted your suggestion and did "commit suggestion", but when I tried to push changes, I get: 
```bash
▶ git push origin display-diagram 
To github.com:reshamas/scikit-learn.git
 ! [rejected]            display-diagram -> display-diagram (non-fast-forward)
error: failed to push some refs to 'git@github.com:reshamas/scikit-learn.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
(sklearndev) 
```

Reply:
as suggested in the message, you need a `git pull` ;)

This happens because the branch on your fork (on the cloud) and the one on your local machine have diverged: they both got one commit that the other one doesn't

Reply:
I tried both `git pull` and `git pull upstream master`. I also tried various permutations, and I am now "Lost in Git"

```bash
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=<remote>/<branch> display-diagram

(sklearndev) 
```

Reply:
the issue isn't with `upstream master` (which is this repo's main branch) but with this branch on your own fork: `origin display-diagram`

I'm a bit surprise `git pull` doesn't work, maybe try `git pull origin display-diagram` then?

Reply:
Though it looks like you're on the `sklearndev` branch on your machine? (or is that the name of the virtualenv?)

Reply:
ok, it went through.  (the steps worked, but I needed to commit my changes first)

Reply:
you have merge conflicts that weren't taken out of the file now